public/scripts/main.js

```diff
@@ -81,18 +79,18 @@ document.querySelector('.dark-mode-btn').addEventListener('click', function () {
     document.querySelector('body').classList.toggle('dark-mode');
 });
 
-const makePoo = () => {
-    let rand = Math.round(Math.random() * (10000) + 20000);
-    setTimeout(function() {
+var makePoo = function () {
+    var rand = Math.round(Math.random() * (10000) + 20000);
+    setTimeout(function () {
         makePoo();
-        const poop = document.getElementsByClassName('poop')[0];
-        const displayStyle = window.getComputedStyle(poop).getPropertyValue('display');
+        var poop = document.getElementsByClassName('poop')[0];
+        var displayStyle = window.getComputedStyle(poop).getPropertyValue('display');
         poop.style.display = displayStyle === 'none' ? 'inline' : displayStyle;
     }, rand);
 };
 
-const cleanUpPoo = (poop) => {
-    const cursorType = document.getElementsByClassName('horse')[0].className;
+var cleanUpPoo = function (poop) {
```

Comment:
Reasoning for switching from const to a var?

Reply:
In your code you had mixed var, let and const. I changed them to var to use gulp without mixing things with Babel. It would be good switching everything to ES6